### PR TITLE
Build release

### PR DESCRIPTION
**Pull request by MikhailTryakhov**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json [hunk 1]
```diff
@@ -513,7 +513,7 @@
             "modelAsString": true
           }
         },
-        "negationConditon": {
```
Comment (EvgenyAgafonchikov) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json]: It looks like this shouldn't be fixed: https://github.com/Azure/azure-rest-api-specs/pull/5341#discussion_r263686120
One of CI failures is [related to this](https://dev.azure.com/azure-sdk/public/_build/results?buildId=20119&view=logs&jobId=8d2f9c49-cd83-5a9d-f3fe-2fd651afa742&taskId=4f68196f-0a37-543a-e233-ac6814560e63&lineStart=2004&lineEnd=2004&colStart=1&colEnd=12)

+@azhao155 to confirm we still need to keep this typo to match server side. If no, we need to update examples then.

**Comment by MikhailTryakhov** [comment]
Hey @sergey-shandar looks like there are still problems with CI... can you please help to figure out? 
Everything in required jobs has correctly passed but result is still failed

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json (hunk 1 above)
Comment (MikhailTryakhov) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json]: reverted, will have mistyping saved

**Comment by azhao155** [comment]
The typo is intended, Thanks for taking care of it.

**Review by sergey-shandar — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json (hunk 1 above)
Comment (nschonni) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-02-01/webapplicationfirewall.json]: Why not fix it if the service hasn't been released yet?

**Pull request merged**